import-mailbox-to-gmail.py

```diff
@@ -182,7 +182,8 @@ def process_mbox_files(username, service, labels):
   for root, dirs, files in os.walk(base_path):
     for dir in dirs:
       try:
-        get_label_id_from_name(service, username, labels, dir)
+        labelname = os.path.join(root[len(base_path) + 1:], dir)
```

Comment:
I think you need to sync your branch - this change is already in the master branch: https://github.com/google/import-mailbox-to-gmail/blob/85fc8afe0480c3323026504c5dedb7d6f6c5fd40/import-mailbox-to-gmail.py#L185